<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>1136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="I_LeftRaw">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>350</width>
      <height>220</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="B_LeftImgPickBtn">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>156</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QListWidget" name="CamerList">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>73</y>
      <width>250</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="B_RightImgPickBtn">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>156</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_LRawImg">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>240</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Rectified Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_AvailableImg">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>47</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Images</string>
    </property>
   </widget>
   <widget class="QLabel" name="I_RightRaw">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>350</width>
      <height>220</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_RRawImg">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>493</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Rectified Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="I_GT">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>770</y>
      <width>350</width>
      <height>220</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_GT">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>740</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Ground Truth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_GTPickBtn">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>156</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ground Truth</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_GlobalParams">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>34</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Global Parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_MinDisp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>33</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Disp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_MaxDisp">
    <property name="geometry">
     <rect>
      <x>702</x>
      <y>34</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Disp:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_MinDisp">
    <property name="geometry">
     <rect>
      <x>646</x>
      <y>33</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_MaxDisp">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>34</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_m_u8BlockSize">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>33</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_m_u8BlockSize_half">
    <property name="geometry">
     <rect>
      <x>1148</x>
      <y>34</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Block Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_P2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>35</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_P2">
    <property name="geometry">
     <rect>
      <x>991</x>
      <y>35</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Penalty 2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_P1">
    <property name="geometry">
     <rect>
      <x>928</x>
      <y>34</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_P1">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>34</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Penalty 1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_SGBMDir">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>34</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Census_2">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>34</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SGBM Directions:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_UniqRatio">
    <property name="geometry">
     <rect>
      <x>1788</x>
      <y>36</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_PostProcessing">
    <property name="geometry">
     <rect>
      <x>1531</x>
      <y>36</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Post-Processing</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_UniqRation">
    <property name="geometry">
     <rect>
      <x>1658</x>
      <y>36</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Uniqueness Ratio:</string>
    </property>
   </widget>
   <widget class="Line" name="O_line_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>1451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="O_line_8">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>34</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="I_CT_HD_SGBM">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="I_CT_HD_Diff">
    <property name="geometry">
     <rect>
      <x>1192</x>
      <y>560</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_SGBM">
    <property name="geometry">
     <rect>
      <x>1035</x>
      <y>80</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Semiglobal Block Matching</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="L_SGBM_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>530</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Difference to Ground Truth</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="L_error_7">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>539</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Average pixel error:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="L_error_6">
    <property name="geometry">
     <rect>
      <x>1676</x>
      <y>540</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Run">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>188</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>78</red>
          <green>154</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>117</red>
          <green>231</green>
          <blue>9</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>97</red>
          <green>192</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>77</green>
          <blue>3</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>103</green>
          <blue>4</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>78</red>
          <green>154</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>166</red>
          <green>204</green>
          <blue>130</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>78</red>
          <green>154</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>117</red>
          <green>231</green>
          <blue>9</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>97</red>
          <green>192</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>77</green>
          <blue>3</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>103</green>
          <blue>4</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>78</red>
          <green>154</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>166</red>
          <green>204</green>
          <blue>130</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>78</red>
          <green>154</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>117</red>
          <green>231</green>
          <blue>9</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>97</red>
          <green>192</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>77</green>
          <blue>3</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>103</green>
          <blue>4</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>77</green>
          <blue>3</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>77</green>
          <blue>3</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>78</red>
          <green>154</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>78</red>
          <green>154</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>78</red>
          <green>154</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="P_Update_In">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="Line" name="O_line_9">
    <property name="geometry">
     <rect>
      <x>1639</x>
      <y>34</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="O_line_10">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>20</width>
      <height>691</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="I_CT_HD_Hist">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>560</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="L_SGBM_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost-Function Ploting / Histogram</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="L_Sources">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_Source">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>../Test_data/</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1850</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
